src/components/availability-panel/drag-drop-context/DroppableComponent.tsx

```diff
@@ -1,18 +1,16 @@
 import { FC } from 'react';
-import { Droppable, Draggable } from 'react-beautiful-dnd';
-import classNames from '@/components/availability-panel/drag-drop-context/styles.module.scss';
+import { Droppable } from 'react-beautiful-dnd';
+// import classNames from '@/components/availability-panel/drag-drop-context/styles.module.scss';
 import { droppableComponent } from '@/interfaces/availabilityPanel.type';
+import DraggableComponent from '@/components/availability-panel/drag-drop-context//DraggableComponent';
 
-const imageGenerator = (name: string) => `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${name}/img.png`;
-
-const getItemStyle = (isDragging: boolean, draggableStyle: any) => {
-  const style = {
-    background: isDragging ? '#d1d1d1' : 'white',
-    ...draggableStyle,
-  };
-  return style;
-};
-
+// const getItemStyle = (isDragging: boolean, draggableStyle: any) => {
+//   const style = {
+//     background: isDragging ? '#d1d1d1' : 'white',
+//     ...draggableStyle,
+//   };
+//   return style;
+// };
```

Comment:
re: dead code

Reply:
I did in hurry, I didn't even see the commented code. sorry😅